js/ui/pivot_grid/ui.pivot_grid.headers_area.js

```diff
@@ -17,6 +18,7 @@ const PIVOTGRID_LAST_CELL_CLASS = 'dx-last-cell';
 const PIVOTGRID_VERTICAL_SCROLL_CLASS = 'dx-vertical-scroll';
 const PIVOTGRID_EXPAND_BORDER = 'dx-expand-border';
 
+const isRenovatedScrollable = !!Scrollable.IS_RENOVATED_WIDGET;
```

Comment:
Do we need two ways to check 'isRenovated'?

1. `if(scrollable?.isRenovated()) {`
2. `const isRenovatedScrollable = !!Scrollable.IS_RENOVATED_WIDGET;`

Reply:
Yep, it is convenient for defining in some scenarios